docs/manual/inference.tex

```diff
@@ -214,6 +214,14 @@
 \item If set, the \verb|JAVA8_HOME| environment variable must point to a Java 8 JDK.
 \item If set, the \verb|JAVA11_HOME| environment variable must point to a Java 11 JDK.
 \item \<CHECKERFRAMEWORK> environment variable must point to a built copy of the Checker Framework.
+\item If set, the \verb|DLJC| environment variable must point to a copy of the \<dljc> script
+from \ahref{https://github.com/kelloggm/do-like-javac}{do-like-javac}. (If this variable is not
+set, the WPI scripts will download this dependency automatically.)
+% This next item is true because of the use of ${*@Q}: @Q is a Bash 4.4 feature and causes "bad substitution"
```

Comment:
I suggest placing a run-time check in the script, so that the manifestation of the problem is less obscure.  This text could be moved there.